src/js/Content/Features/Store/Common/FCustomizer.js

```diff
@@ -30,7 +31,32 @@ export default class FCustomizer extends Feature {
         if (this.context.type === ContextType.APP) {
             this._customizeAppPage();
         } else if (this.context.type === ContextType.STORE_FRONT) {
-            this._customizeFrontPage();
+
+            Messenger.onMessage("renderComplete").then(() => { this._customizeFrontPage(); });
+
+            /*
+             * Run our customizer when GHomepage.bInitialRenderComplete is set to `true`
+             * https://github.com/SteamDatabase/SteamTracking/blob/d22d8df5db80e844f7ae1157dbb8b59532dfe4f8/store.steampowered.com/public/javascript/home.js#L432
+             */
+            Page.runInPageContext(() => {
+
+                // eslint-disable-next-line no-undef
+                if (GHomepage.bInitialRenderComplete) {
+                    window.Messenger.postMessage("renderComplete");
+                    return;
+                }
+
+                // eslint-disable-next-line no-undef
+                GHomepage = new Proxy(GHomepage, {
+                    set(target, prop, value) {
+                        if (prop === "bInitialRenderComplete" && value === true) {
+                            window.Messenger.postMessage("renderComplete");
+                        }
+
+                        return Reflect.set(...arguments);
```

Comment:
Dunno why eslint complains about this...

Reply:
Because you are using the `arguments` object instead of rest parameters